bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelSimuConnector.java

```diff
@@ -44,6 +44,7 @@
 public class RotelSimuConnector extends RotelConnector {
 
     private static final int STEP_TONE_LEVEL = 1;
+    private static final double STEP_DECIBEL = 0.5;
```

Comment:
Is this consistent across all models and zones and non-configurable on the receiver? For Denon/Marantz step is usually 0.5 dB for main zone and 1 dB for other zones.

Reply:
This is not the general volume but the adjustment on each channel in a multi channel processor. +/- 0.5 dB is what is mentioned in the API. The API only provides UP and DOWN commands for each channel.
Please note that this constant is defined and only used in my simulator class, so this is not really important.
Even if a real model would use a different step (not the one defined in the Rotel API !), this will change absolutely nothing in the binding. And the resulting value is returned by the API in response to the UP/DOWN command.
These commands are available today only for 3 models: RSP-1576, RSP-1582 and RAP-1580.

Reply:
In that PR, I just made these existing API commands available to the user, this is a minimal implementation without any display of the current value.

In the future, I could consider adding an OH channel for each of them but I miss an API command to get the current value. I could send UP and then DOWN to retrieve the current value. I just don't know if these commands are always available.